### PR TITLE
added response and these files and folders to assessment

### PR DESCRIPTION
**Pull request by resource11**

### CODE REVIEW

File: app/models/licensee.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Licensee < ActiveRecord::Base
+end
```
Comment (GA-MEB) [inline app/models/licensee.rb:2]: Model : check.

File: db/migrate/20151029153137_create_licensees.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      t.string :height
+      t.string :weight
+      t.string :dob
+      t.string :eye_color
+      t.string :hair_color
+
+      t.timestamps null: false
+    end
+  end
+end
```
Comment (GA-MEB) [inline db/migrate/20151029153137_create_licensees.rb:16]: Migration : check.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    t.string   "height"
+    t.string   "weight"
+    t.string   "dob"
+    t.string   "eye_color"
+    t.string   "hair_color"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+end
```
Comment (GA-MEB) [inline db/schema.rb:32]: Ran `rake db:migrate` : check.

File: assessment.rb [hunk 4]
```diff
@@ -1,12 +1,61 @@
...
 
-# question 3:
 
-# question 4:
 
-# question 5:
+# Question 2. Use ActiveRecord to add Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black into the table of licensees.
+# in the  scripts/create_licensee.rb file
+licensee = Licensee.create(surname: 'Bishop', given_name: 'Caren', gender: 'f', height: 63, weight: 132, dob: '1943-09-26', eye_color: 'Brown', hair_color: 'Black')
+p licensee
```
Comment (GA-MEB) [inline assessment.rb:11]: Correct.

File: assessment.rb [hunk 5]
```diff
@@ -1,12 +1,61 @@
...
+          next if Licensee.exists? licensee
+          Licensee.create! licensee
+        end
+      end
+    end
+# end licensee table section
+  end
+end
+
+
```
Comment (GA-MEB) [inline assessment.rb:35]: Correct.

File: assessment.rb [hunk 6]
```diff
@@ -1,12 +1,61 @@
...
+# Question 4. Use ActiveRecord to retrieve all the licensees with Hazel eye color and Brown or Black hair color.
+
+# still shaky on syntax
+eye_hair = Licensee.find_by! eye_color: 'Hazel', hair_color: 'Brown' or hair_color 'Black'
+
```
Comment (GA-MEB) [inline assessment.rb:40]: This syntax is a little off. Did you try running it?

File: assessment.rb [hunk 7]
```diff
@@ -1,12 +1,61 @@
...
+# Question 5. Use ActiveRecord to make each of Marilynn Escobar and Chris Whaley one inch shorter. Look them up by their names.
+
+# still shaky on syntax
+Licensee.transaction do
+  licensees = Licensee.all
+  licensees.each do |licensee|
+    licensee.height -= 1 if licensee.find_by(surname:) == 'Escobar' licensee.find_by(given_name:) == 'Marilynn'
```
Comment (GA-MEB) [inline assessment.rb:47]: You're calling `.find_by` on an instance of the model, but it's only available as a method on the model class. It's also not necessary here - you can just refer to the properties directly (i.e. `licensee.surname == ...`)

File: assessment.rb [hunk 8]
```diff
@@ -1,12 +1,61 @@
...
+    licensee.height -= 1 if licensee.find_by(surname:) == 'Whaley' licensee.find_by(given_name:) == 'Chris'
+    licensee.save!
```
Comment (GA-MEB) [inline assessment.rb:49]: Otherwise correct.

File: assessment.rb [hunk 9]
```diff
@@ -1,12 +1,61 @@
...
+
+# Question 6. Use ActiveRecord to remove Dylan Rich and Teresita Myers from the table of licensees.
+
+dylan_rich = Licensee.find_by! surname: 'Rich', given_name: 'Dylan'
+
+dylan_rich.destroy!
+
+teresita_myers = Licensee.find_by! surname: 'Myers', given_name: 'Teresita'
+
+teresita_myers.destroy!
```
Comment (GA-MEB) [inline assessment.rb:61]: Correct.

**Comment by GA-MEB** [comment]
4.75 of 6 (-0.75 on Q4, -0.5 on Q5)

Very nice work, Kathleen.

**Comment by resource11** [comment]
Woo! Will look at what I missed so I can apply this well in project 2!

> ^.^< 
> Sent from my katPhone
>

**Pull request closed without merging**